jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerClient.java

```diff
@@ -140,11 +141,10 @@ public String load(ImageTarball imageTarball, Consumer<Long> writtenByteCountLis
       try (InputStreamReader stderr =
           new InputStreamReader(dockerProcess.getErrorStream(), StandardCharsets.UTF_8)) {
         error = CharStreams.toString(stderr);
-
       } catch (IOException ignored) {
-        // This ignores exceptions from reading stderr and throws the original exception from
+        // This ignores exceptions from reading stderr and uses the original exception from
         // writing to stdin.
-        throw ex;
+        error = ex.getMessage();
```

Comment:
Couldn't this code use the same `getStderrOutput(dockerProcess))` as below?

Reply:
I actually tried that and contemplated the code. Went back and forth. `getStderrOutput()` returns the canned `unknown` message when an IO error occurs while reading the stderr message. However, the original code wants that if such an IO error occurs, it shows the error message of the original IO exception that occurred while writing into the process, which reveals more useful information. I could change `getStderrOutput()` to accept a "default" error message to print if it fails to read stderr for any reason, but since here is the only usage for that, so I decided to leave it read stderr manually here.

Reply:
That makes sense; would be worth a comment to that effect here